exe/Parser/Text.hs

```diff
@@ -529,32 +529,32 @@ genModules indir outdir props = do
           & Stream.fold (Fold.mkPureId (++) [])
 
     let compositions =
-            ( "Unicode.Properties.Compose"
+            ( "Unicode.Internal.Generated.UnicodeData.Compositions"
```

Comment:
Why do we need `UnicodeData`? Can we avoid that to keep the name shorter?

Reply:
It's because this module is `Generated` from `UnicodeData.txt`

Reply:
We should remove it though. It shouldn't really matter where it's generated from anyway.

Reply:
Just like `Generated.DerivedCoreProperties` is generated from `DerivedCoreProperties.txt`.

Reply:
These are generated modules anyway and the users should not directly use these.

Reply:
Can we name the source of the file in the generated files e.g.:

```
 -- autogenerated from ucd/DerivedCoreProperties.txt 
 ```